### PR TITLE
Add documentation about lighttpd usage

### PR DESCRIPTION
**Pull request by jronak**
Added instructions for installing and running cgimap with lighttpd

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -82,8 +82,15 @@ Setup
...
-on the command line with a command like
+CGImap with lighttpd you will need to install lighttpd:
+
+	sudo apt-get install lighttpd
+
+Edit the supplied lighttpd.config file to include your CGImap path and run it with the lighttpd like
+
+	/usr/sbin/lighttpd -f lighttpd.conf
+
+Run CGImap on the command line with a command like
```
Comment (pnorman) [inline README]: Didn't we conclude that lighttpd handles starting cgimap and if you're using the config file you don't need to also start the binary?

Comment (jronak) [inline README]: Yes, FastCGI handles starting the binary, but I thought we are starting the binary for debugging?

Comment (pnorman) [inline README]: If lighttpd starts the binary then there's no purpose running a second copy

Comment (pnorman) [inline README]: Note: we still need to document how to start the binary, just make it clear you don't do that when running it through lighttpd with the supplied config file

Comment (jronak) [inline README]: Updated PR, please review.

**Comment by zerebubuth** [comment]
Thanks!

**Pull request merged**